### DISCUSSION TITLE
关于openclash的功能疑问

### DISCUSSION THREAD

**Discussion by houzi-** [vernesong/OpenClash#2503] [Q&A]
请问`/etc/openclash/rule_provider`里的规则集默认启动就加载的吗？是不是这些规则集放在启动配置文件目录下就会识别加载，不需要另外去设置的
![屏幕截图 2022-06-20 162548](https://user-images.githubusercontent.com/9395995/174558758-133886b9-5b09-42ec-aeb1-c72dfbec1fae.jpg)

**Comment by kingsey** [comment]
需要做配置，用RULE-SET和rule-providers，具体参考clash的wiki，openclash也做了相关页面，但说实话我觉得不如直接写配置好用。

**Reply by KazeLiu** [reply]
如果我想使用rule-providers，那么我应该如何操作使它能在更新订阅后自动追加？我不太会用。谢谢

**Comment by houzi-** [comment]
好的👌，感谢指导
